### PR TITLE
Adds the Brig Phys

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Not much to say. They have all the stuff they do on Paradise, plus a fancy cap and a fancy labcoat.

Also, all labcoats can now carry full-size flashlights instead of just penlights. I have nothing but my personal fondness of flashlights to justify this with.

### CODE REVIEW

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -105,6 +105,15 @@
...
 	item_state = "greydet"
 
+//Brig Phys
+/obj/item/clothing/suit/hazardvest/brig_phys
+	name = "brig physician's vest"
+	desc = "A lightweight vest worn by the Brig Physician."
+	icon_state = "brig_phys_vest"
+	item_state = "sec_helm"//looks kinda similar, I guess
+	allowed = list(/obj/item/device/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/device/healthanalyzer, /obj/item/device/flashlight, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic, /obj/item/soap, /obj/item/device/sensor_device, /obj/item/tank/internals/emergency_oxygen)
+	armor = list(melee = 15, bullet = 15, laser = 15,energy = 10, bomb = 10, bio = 50, rad = 10, fire = 50, acid = 50)
```
Comment (SamHPurp) [inline code/modules/clothing/suits/jobs.dm]: 🤔 that isn't bad stats, is this what it is on Para atm?

Comment (Birdtalon) [reply code/modules/clothing/suits/jobs.dm]: Yeah these stats are far too high.

Here are Paradise's for comparison.
`armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0)`

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -21,6 +21,12 @@
 	icon_state = "labcoat_emt"
 	item_state = "labcoat_cmo"
 
+/obj/item/clothing/suit/toggle/labcoat/brig_phys
+	name = "security medic's labcoat"
+	icon_state = "labcoat_sec"
+	item_state = "labcoat_sec"
+	armor = list(melee = 10, bullet = 10, laser = 10,energy = 10, bomb = 10, bio = 50, rad = 0, fire = 50, acid = 50)
```
Comment (Birdtalon) [inline code/modules/clothing/suits/labcoat.dm]: Stats here too.

File: code/game/objects/effects/landmarks.dm [hunk 3]
```diff
@@ -132,6 +132,9 @@
 /obj/effect/landmark/start/blueshield
 	name = "Blueshield"
 
+/obj/effect/landmark/start/brig_phys
```
Comment (FlattestGuitar) [inline code/game/objects/effects/landmarks.dm]: It's here, just not on any of the maps.

Comment (SamHPurp) [reply code/game/objects/effects/landmarks.dm]: oh sorry I missed it. soz bruv

**Pull request merged**